crates/network-scanner-net/src/runtime.rs

```diff
@@ -40,7 +41,7 @@ impl Drop for Socket2Runtime {
     }
 }
 
-const QUEUE_CAPACITY: usize = 1024;
+const QUEUE_CAPACITY: usize = 8024;
```

Comment:
Maybe too big?

Reply:
Any reason why you changed?

Reply:
yes, it was causing errors on registering the events.